### DISCUSSION TITLE
in Zed how to 'run code' with a keyboard shortcut.

### DISCUSSION THREAD

**Discussion by SnapshotCiTy** [zed-industries/zed#13982] [Feature Requests]
Using Zed to programm in Golang.
At the left to the main function, there is a 'play' button to execute the code, which is great and faster than doing it through the terminal. 

I didn't find a keyboard shortcut for this or the way to define a custom keybinding for it.

Is there a way?

**Comment by SomeoneToIgnore** [comment]
There's an 
`"cmd-.": "editor::ToggleCodeActions",` default macOs binding that will either spawn a test, if it's the only action that could be run at the current caret, or will open the menu like
![image](https://github.com/zed-industries/zed/assets/2690773/d6f75867-d895-48cf-aa00-fb5eb13d9371)

to select a task to run, including the test.
